src/components/SingleName/NameRegister/Explainer.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import moment from 'moment'
+
+import mq from 'mediaQuery'
+import Step from './Step'
+import Button from '../../Forms/Button'
+import { ReactComponent as Bell } from '../../Icons/Bell.svg'
+import { ReactComponent as Tick } from '../../Icons/GreyCircleTick.svg'
+
+import { requestPermission, hasPermission } from './notification'
+
+const Steps = styled('section')`
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-row-gap: 30px;
+  ${mq.large`
+    grid-template-columns: repeat(3, 1fr);
+    grid-column-gap: 30px;
+    grid-template-rows: 1fr;
+  `}
+`
+
+const Header = styled('div')`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+
+  h2 {
+    font-family: Overpass;
+    font-weight: 300;
+    font-size: 18px;
+    color: #2b2b2b;
+    letter-spacing: 0;
+    margin: 0;
+    margin-bottom: 5px;
+
+    ${mq.medium`
+      font-size: 24px;
+      color: #2B2B2B;
+      letter-spacing: 0;
+    `}
+  }
+
+  p {
+    margin: 0;
+    font-weight: 400;
+    font-family: Overpass;
+    font-size: 14px;
+    color: #adbbcd;
+    letter-spacing: 0;
+  }
+`
+
+const NotifyButton = styled(Button)`
+  flex-shrink: 0;
+`
+
+const NotifyButtonDisabled = styled('div')`
+  color: hsla(0, 0%, 82%, 1);
+`
+
+const Explainer = ({ step, waitPercentComplete, waitTime }) => {
+  const titles = {
+    PRICE_DECISION: 'Registering a name requires you to complete 3 steps',
+    COMMIT_SENT:
+      'Don’t close your browser! You’ll be able to register your name soon.',
+    COMMIT_CONFIRMED:
+      'Don’t close your browser! You’ll be able to register your name soon.',
+    AWAITING_REGISTER:
+      'Don’t close your browser! You’ll be able to register your name soon.',
+    REVEAL_SENT:
+      'Don’t close your browser! You’ll be able to register your name soon.',
+    REVEAL_CONFIRMED: 'You’ve completed all the steps, manage your name now!'
+  }
+
+  return (
+    <>
+      <Header>
+        <div>
+          <h2>{titles[step]}</h2>
+          <p>
+            *Favorite the name for easy access in case you close out of your
+            browser.
+          </p>
+        </div>
+        {hasPermission ? (
+          <NotifyButtonDisabled>
+            <Tick style={{ marginRight: 5 }} />
+            Notify me
+          </NotifyButtonDisabled>
+        ) : (
+          <NotifyButton type="hollow-primary" onClick={requestPermission}>
```

Comment:
This one is always greyed out when I test .

<img width="155" alt="Screenshot 2019-04-01 at 19 46 31" src="https://user-images.githubusercontent.com/2630/55351702-e9837b80-54b6-11e9-9a0f-85981685db50.png">

Reply:
Ah this needs to be fixed, i'll need to look into this. Will make an issue